stdlib/p7zip_jll/src/p7zip_jll.jl

```diff
@@ -85,6 +85,8 @@ function __init__()
     init_p7zip_path()
     PATH[] = dirname(p7zip_path)
     push!(PATH_list, PATH[])
+    append!(LIBPATH_list, [joinpath(Sys.BINDIR, Base.LIBDIR, "julia"), joinpath(Sys.BINDIR, Base.LIBDIR)])
+    LIBPATH[] = join(LIBPATH_list, pathsep)
```

Comment:
As far as I could see, most JLLs do something like
https://github.com/JuliaLang/julia/blob/5d7e13f0dec118f86df2cf3d8999549cd47c32c2/stdlib/LibGit2_jll/src/LibGit2_jll.jl#L33-L34
because they're libraries, and only need to add to the libpath the directory they're in.  However, 7zip is in a different directory, the best I could do was to try both `joinpath(Sys.BINDIR, Base.LIBDIR, "julia")` and `joinpath(Sys.BINDIR, Base.LIBDIR)`, is there a better way to find the directory where libstdc++ is?